packages/competence/.env

```
@@ -5,3 +5,4 @@ TI_INSTANCE_NAME=ti-competence
 TI_LOCALIZATION_LABELS_PATH=bin/localization/competence-labels.json
 TI_WEB_APP_STATIC_CACHE_DISABLED=true
 COMPETENCE_PRELOAD_DATA=false
+COMPETENCE_TEST_USER_ENABLED=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The COMPETENCE_TEST_USER_ENABLED key should go before the TI_AUDITING_LOG_MIN_LEVEL key

(UnorderedKey)